pages/linux/protontricks.md

```diff
@@ -18,3 +18,11 @@
 - Run a command within a game's installation directory:
 
 `protontricks -c {{command}} {{appid}}`
+
+- List all installed games:
+
+`protontricks -l`
+
+- Search for a game's App ID by name:
+
+`protontricks -s {{Game_Name}}`
```

Comment:
```suggestion
`protontricks -s {{game_name}}`
```

Just snake_case, nothing more is permitted unless it's a placeholder sample value.